torch/_export/exported_program.py

```diff
@@ -166,10 +191,106 @@ def transform(self, *passes: PassType) -> "ExportedProgram":
         transformed_ep.graph_module.meta.update(res.graph_module.meta)
         return transformed_ep
 
-    def _add_runtime_assertions(self) -> "ExportedProgram":
-        return self.transform(
-            _AddRuntimeAssertionsForConstraintsPass(self.range_constraints, self.equality_constraints)
+    def _add_runtime_assertions(
+        self,
+        functionalize: bool,
+    ) -> "ExportedProgram":
+        ep = self.transform(
+            _AddRuntimeAssertionsForConstraintsPass(
+                self.range_constraints,
+                self.equality_constraints,
+            )
+        )
+        # Graph signature update should be part of pass run instead of a
+        # separate step. However this requires augmenting pass infra at fx level
+        # to operate on `ExportedProgram` instead of `fx.GraphModule`.
+        # TODO: Integrate graph signature update into pass run.
+        ep = _update_graph_signature_after_adding_runtime_assertions(
```

Comment:
If I remember correctly, this can happen after any pass right? Should we rename this to something more generic? (like fixup_graph_signature)

Reply:
Will handle in a separate small patching PR :)